nix/overlay.nix

```diff
@@ -69,7 +69,7 @@ in {
   yarn = super.yarn.override { nodejs = super.nodejs-18_x; };
   openjdk = super.openjdk11_headless;
   xcodeWrapper = super.xcodeenv.composeXcodeWrapper {
-    version = "14.0";
+    version = "15.1";
```

Comment:
We'll need to perform this upgrade separately.

Reply:
This has already been done:

* https://github.com/status-im/status-mobile/pull/18401